### DISCUSSION TITLE
analysing graph plotted for the benchmark performance

### DISCUSSION THREAD

**Discussion by dipshirajput** [qdrant/qdrant#3937] [Q&A]
Hi all,
i was trying to benchmark qdrant and am able to generate a graph as shown in the attached screenshot.
please let me know how can i analyse it ?
![qdr](https://github.com/qdrant/qdrant/assets/52161957/290df15a-53e4-4d72-bb1d-11dbc29ae6b2)

Thanks in advance.

**Comment by generall** [comment]
recall values below 0.9 are useless, you can safely ignore them. The interesting part is on  the 0.9-1.0 range.
